backend/internal/api/api.go

```diff
@@ -229,5 +222,11 @@ func (h Handler) DBPostHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	fmt.Fprint(w, string(jsonID))
-	w.WriteHeader(http.StatusOK)
+
+	event := events.ImgEvent{Img: img}
+	eventHandler := events.NewEventHandler(&event, utils.NewIdGenerator())
+	err = eventHandler.SendEvent()
```

Comment:
The implementation is incomplete. I run the test without setting `EVENT_URL` and it was green. If error is returned from `SendEvent` function  handler should return 5xx http status as logging is not enough.